docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Pod Lifecycle
+---
+
+{% capture overview %}
+
+{% comment %}Updated: 4/14/2015{% endcomment %}
+{% comment %}Edited and moved to Concepts section: 2/2/17{% endcomment %}
+
+This page describes the lifecycle of a Pod.
+
+{% endcapture %}
+
+
+{% capture body %}
+
+## Pod phase
+
+A Pod's `status` field is a
+[PodStatus](/docs/resources-reference/v1.5/#podstatus-v1)
+object, which has a `phase` field.
+
+The phase of a Pod is a simple, high-level summary of where the Pod is in its
+lifecycle. The phase is not intended to be a comprehensive rollup of observations
+of Container or Pod state, nor is it intended to be a comprehensive state machine.
+
+The number and meanings of Pod phase values are tightly guarded.
+Other than what is documented here, nothing should be assumed about Pods that
+have a given `phase` value.
+
+Here are the possible values for `phase`:
+
+* Pending: The Pod has been accepted by the Kubernetes system, but one or more of
+  the Container images has not been created. This includes time before being
+  scheduled as well as time spent downloading images over the network,
+  which could take a while.
+
+* Running: The Pod has been bound to a node, and all of the Containers have been
+  created. At least one Container is still running, or is in the process of
+  starting or restarting.
+
+* Succeeded: All Containers in the Pod have terminated in success, and will not
+  be restarted.
+
+* Failed: All Containers in the Pod have terminated, and at least one Container
+  has terminated in failure. That is, the Container either exited with non-zero
+  status or was terminated by the system.
+
+* Unknown: For some reason the state of the Pod could not be obtained, typically
+  due to an error in communicating with the host of the Pod.
+
+## Pod conditions
+
+A Pod has a PodStatus, which has an array of
+[PodConditions](docs/resources-reference/v1.5/#podcondition). Each element
+of the PodCondition array has a `type` field and a `status` field. The `type`
+field is a string, with possible values PodScheduled, Ready, Initialized, and
+Unschedulable. The `status` field is a string, with possible values True, False,
+and Unknown.
```

Comment:
Hi!
Sorry to bother 😁 I traced the addition of the 'Unschedulable' condition type to this commit.
But I can't find anywhere where the condition 'Unschedulable' is set (either in the kubelet or in the scheduler.go).

Is 'Unschedulable' just a reason of PodScheduled?
Or is it also a condition type?